build-release.sh

```diff
@@ -60,16 +60,16 @@ checksum() {
 	local CHECKSUM_FILE="k6-${VERSION}-checksums.txt"
 
 	if command -v sha256sum > /dev/null; then
-		CHECKSUM_CMD="sha256sum"
+		CHECKSUM_CMD=("sha256sum")
 	elif command -v shasum > /dev/null; then
-		CHECKSUM_CMD="shasum -a 256"
+		CHECKSUM_CMD=("shasum" "-a" "256")
 	else
 		echo "ERROR: unable to find a command to compute sha-256 hash"
 		return 1
 	fi
 
 	rm -f "dist/$CHECKSUM_FILE"
-	( cd dist && for x in *; do "$CHECKSUM_CMD" "$x" >> "$CHECKSUM_FILE"; done )
+	( cd dist && for x in ./*; do "${CHECKSUM_CMD[@]}" "$x" >> "$CHECKSUM_FILE"; done )
```

Comment:
`./*` would produce different outputs than the current checksum files: https://github.com/loadimpact/k6/releases/download/v0.25.1/k6-v0.25.1-checksums.txt

Reply:
Ah right, because the filename is changed

Reply:
Fixed by using `--` instead.

Reply:
Hmm, oh wait, the difference because we now use `CGO_ENABLED=0`? The filename should not affect the checksum of file content right?

Reply:
@na-- Also, since we don't use `-trimpath`, so the binary output will not be deterministic.

Reply:
Ah, knew I forgot something from Go 1.13! Yeah, we should definitely use `-trimpath` everywhere we build packages! Can you add it?

Regarding what I originally said about `./*`, this is the difference. Without it, i.e. with just a star glob, you'd get a `checksum.txt` file like this (i.e. like all previous checksum files):
```
7c71fbad3f1562030761447bf822e13c556b2e0389c2c0676f06d9bb73394f24  k6-v0.25.1-linux32.tar.gz
5b5a2273c215f50ad29fb7a3beed86c420ee2f683bede15c72cd5a939fc79d75  k6-v0.25.1-linux64.tar.gz
a0c29f05042f78314ef94957e03be4b9192861d42749b2811ca3ca260601eb90  k6-v0.25.1-mac.zip
1641184e540b45c243ea2d9fa18897d98a980a3c623131ab0afcd60fdd773a3b  k6-v0.25.1-win32.zip
3ba7999afda2bb79167765fd91b1c61579e38bbf9143de09f12f35c6edd53ea0  k6-v0.25.1-win64.zip
```
with `./*` you'd get:
```
8f173a1f10f08c3927a4f26ada33905c6d374557c92a3e663f67b5ddc4f51439  ./k6-v0.25.1-linux32.tar.gz
673ad4d32e0502746c6bed1549175f7718913da671637d6c1e70004fcb3dd05d  ./k6-v0.25.1-linux64.tar.gz
19e36f2539f8029c1b0a63acb5a8c139aa1c263a1ea9d3521e9e25e9bc9fa62a  ./k6-v0.25.1-mac.zip
2ae87553f835c22d9e62ca2edd085b18d9cac8cb451c7ae1b129649800aca733  ./k6-v0.25.1-win32.zip
2253ee5ff5e61b8c645f00eae4396fe27c1b0196dac2b1e020cc7d6cc9293339  ./k6-v0.25.1-win64.zip
```

notice the needless `./` everywhere - not a big deal, but a needless change

Reply:
@na-- Even with `-trimpath`, the binary is still reproducible between machines, we also need to set `buildid` IIRC.

Reply:
See https://github.com/golang/go/issues/35435

Reply:
Oh well :disappointed: Still, reproducibility can wait until Go 1.14, its lack is not a huge deal right now. My main desire right now is to remove the `GOPATH`s in any stacktraces of k6 panics, which `-trimpath` would accomplish even now. 

You can see all sorts of different home folder paths in a lot of old reports of panics (https://github.com/loadimpact/k6/issues/784, https://github.com/loadimpact/k6/issues/693 and a lot of others), which is a bit embarrassing... Also, the paths were different, depending on whether the version was built in CircleCI or AppVeyor or with the build script...

Reply:
Fixed